packages/discussions/graphql/resolvers/_queries/comments.js

```diff
@@ -19,23 +19,33 @@ module.exports = async (_, args, context, info) => {
     orderBy = 'publishedAt',
     orderDirection = 'DESC',
     first: limit = 40,
-    offset = 0
+    offset = 0,
+    discussionId
   } = options
 
   if (limit > MAX_LIMIT) {
     throw new Error(t('api/discussion/args/first/tooBig', { max: MAX_LIMIT }))
   }
 
-  const sortKey = getSortKey(orderBy)
+  let sortKey = getSortKey(orderBy)
+  if (sortKey === 'score') {
+    sortKey = '"upVotes" - "downVotes"'
```

Comment:
Would either change code in `lib/sortKey` to return `"upVotes" - "downVotes"` (not sure about its ramifications), or not overwrite `sortKey` and create a new variable, as to not confuse flow here.